map/src/test/java/openlr/map/simplemockdb/SimpleMockedException.java

```diff
@@ -0,0 +1,7 @@
+package openlr.map.simplemockdb;
+
+public class SimpleMockedException extends RuntimeException {
+    public SimpleMockedException(String errorMessage) {
+        super("Simple mocked database exception:" + errorMessage);
```

Comment:
I think the string prefix "Simple mocked database exception:" is reduntant.  This is clear from the exception type.  You can just use `super(errorMessage)`

Reply:
9921bc46eaef1f2b022379c9e24cba52db5b1735